### PR TITLE
docs update for cri-containerd to cri move

### PR DESCRIPTION
**Pull request by mikebrow**
Signed-off-by: Mike Brown <brownwm@us.ibm.com>

### CODE REVIEW

**Comment by Random-Liu** [comment]
Will try to get to this tonight.

**Comment by mikebrow** [comment]
no rush it needs another pass and I need to include your prior comments.. and include couple recent updates

File: docs/testing.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-CRI-Containerd Testing Guide
-============================
-This document assumes you have already setup the development environment (go, git, `cri-containerd` repo etc.).
- 
+========================
```
Comment (Random-Liu) [inline docs/testing.md]: Remove this line. Preview the document you'll see that.

File: docs/installation.md [hunk 2]
```diff
@@ -40,29 +39,29 @@ $ tar -tf cri-containerd-1.0.0-beta.0.linux-amd64.tar.gz
...
 ### Binary Information
 Information about the binaries in the release tarball:
 
-|           Binary Name          |      Support      |   OS  | Architecture |
-|:------------------------------:|:-----------------:|:-----:|:------------:|
-|          cri-containerd        | seccomp, apparmor | linux |     amd64    |
-|               runc             | seccomp, apparmor | linux |     amd64    |
-|   containerd/containerd-shim   |   overlay, btrfs  | linux |     amd64    |
+|           Binary Name          |      Support       |   OS  | Architecture |
+|:------------------------------:|:------------------:|:-----:|:------------:|
```
Comment (Random-Liu) [inline docs/installation.md]: We should merge these 2 rows. I can address this in a latter PR.

File: docs/testing.md (hunk 1 above)
Comment (mikebrow) [reply docs/testing.md]: done

**Comment by mikebrow** [comment]
ok... done for now I modified the following in registry.md "The endpoint is a list that can contain multiple image registry URLs splited by commas." changed to splited to split...

**Comment by mikebrow** [comment]
docs are all yours heading out to dinner cheers

**Pull request merged**